<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label_30">
     <property name="text">
      <string>Est Time</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_31">
     <property name="text">
      <string>Classification Progress</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_32">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="RunTree" name="runTree">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>Samples</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="toolTip">
      <string/>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="toolTip">
      <string>Run the MARCO model on the currently selected run. MARCO has been trained on visible light images and will only run on runs photographed using standard visible light imaging.
</string>
     </property>
     <property name="text">
      <string>Classify Selected Run</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RunTree</class>
   <extends>QTreeWidget</extends>
   <header>polo/widgets/run_tree</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
